Bottom: 733bfae67fc7f4b17b88aa8d59dcf45f19cb3218
Top:    9fc678ad233ed078aa6dd70b1fdd08246a266644
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 20:14:46 +0000

nelma: Fix prototype on set_layer() method to include the "..." flag.


---

diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index e7d0f9f..2180325 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -698,7 +698,7 @@ nelma_do_export(HID_Attr_Val * options)
 /* *** PNG export (slightly modified code from PNG export HID) ************* */
 
 static int 
-nelma_set_layer(const char *name, int group)
+nelma_set_layer(const char *name, int group, int empty)
 {
 	int             idx = (group >= 0 && group < max_layer) ?
 	PCB->LayerGroups.Entries[group][0] : group;
